Dockerfile

```diff
@@ -3,11 +3,17 @@ FROM docker.io/node:22-alpine
 ENV TZ=Europe/Berlin
 EXPOSE 3033
 
-RUN apk add --no-cache autoconf automake build-base make nasm zlib-dev python3 py3-setuptools
+RUN apk add --no-cache \
+    autoconf \
+    automake \
+    build-base \
+    make \
+    nasm \
+    python3
```

Comment:
Die packages zlib-dev und py3-setuptools sind nicht mehr notwendig?

Reply:
builds and works without them, py3-setuptools are definitely not required, those dependencies are related to some css trnasipiling thing ... and that still works